integration/src/main/java/scripts/testscripts/UpdateReferenceResources.java

```diff
@@ -102,18 +104,15 @@ protected void doCleanup(List<TestUserSpecification> testUsers, WorkspaceApi wor
   }
 
   @Override
-  public void setParameters(List<String> parameters) throws Exception {
-    // TODO: Refactor this function when TestRunner starts supporting parameterMap
+  public void setParameters(Map<String, String> parameters) throws Exception {
     super.setParameters(parameters);
-    if (parameters == null || parameters.size() < 4) {
+    if (parameters == null
```

Comment:
I wonder if we could have some util methods to handle these and DRY it up a bit. This is similar to the FlightMap as an anonymous map, so two things that might be useful:
1. Something like the FlightUtil Jay wrote to test that all expected keys are present in the map
2. Something that does "get key XYZ and throw if you don't find it"
I think that would make the code more readable

Reply:
Added suggestion 2, that cleans up the configs nicely. 

I don't think we need 1 since calling "get key X or throw" for each param we need already validates keys before the tests run. Including the full map check just meant each base test script had a "check for all keys or throw" line, then the "get key or throw" lines,  which seemed redundant.